src/diagonal.works/b6/ingest/gtfs/gtfs.go

```diff
@@ -0,0 +1,294 @@
+package gtfs
```

Comment:
i removed timeweights logic from this file / and relying on graph.transittimeweights / graph/tests